crates/release_plz/src/changelog_config.rs

```diff
@@ -37,6 +37,7 @@ impl ChangelogCfg {
 }
 
 /// Used for modifying commit messages.
+// TODO: why re-define this instead using it from git-cliff?
 #[derive(Serialize, Deserialize, Default, PartialEq, Eq, Debug, Clone, JsonSchema)]
```

Comment:
these structs don't have all the derives release-plz needs. That's the main reason why I duplicated them.
E.g. TextProcessor has `#[derive(Debug, Clone, Serialize, Deserialize)]`. 
git-cliff TextProcessor has `pattern: Regex`, which makes it difficult to implement `Eq` for example.

Reply:
Got it, I'm down to implement the necessary traits for these types!

Reply:
> I'm down to implement the necessary traits for these types!

Definitely not urgent. Plus, there are some pros to have these structs in release-plz. E.g. I can document them in the context of release-plz and have the json schema update accordingly.